src/main/java/uk/gov/hmcts/darts/task/runner/impl/ProcessDetsToArmResponseAutomatedTask.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.darts.task.runner.impl;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.arm.component.AutomatedTaskProcessorFactory;
+import uk.gov.hmcts.darts.common.repository.AutomatedTaskRepository;
+import uk.gov.hmcts.darts.log.api.LogApi;
+import uk.gov.hmcts.darts.task.api.AutomatedTaskName;
+import uk.gov.hmcts.darts.task.config.AutomatedTaskConfigurationProperties;
+import uk.gov.hmcts.darts.task.runner.AutoloadingManualTask;
+import uk.gov.hmcts.darts.task.service.LockService;
+
+import static uk.gov.hmcts.darts.task.api.AutomatedTaskName.PROCESS_DETS_TO_ARM_RESPONSE;
+
+@Slf4j
+@Component
+public class ProcessDetsToArmResponseAutomatedTask extends AbstractLockableAutomatedTask
+    implements AutoloadingManualTask {
+
+    @Autowired
+    public ProcessDetsToArmResponseAutomatedTask(AutomatedTaskRepository automatedTaskRepository,
+                                                 AutomatedTaskConfigurationProperties automatedTaskConfigurationProperties,
+                                                 AutomatedTaskProcessorFactory automatedTaskProcessorFactory,
+                                                 LogApi logApi, LockService lockService) {
+        super(automatedTaskRepository, automatedTaskConfigurationProperties, logApi, lockService);
+    }
+
+    @Override
```

Comment:
I think you also need these changes (see reference PR)

<img width="1234" alt="image" src="https://github.com/user-attachments/assets/acca42ea-cb4a-4efd-89c0-2bc034e0795f">

Reply:
I did some work last week to tidy this logic up to reduce boiler plate so it gets automatically created now :)